### PR TITLE
Component-Aaron: Referral

### PR DESCRIPTION
**Pull request by twikista**
Added 
Referral component GIF
Referral component

### CODE REVIEW

**Comment by twikista** [comment]
Refactored referral component into a client side component since it has click interactions

File: Developers/app/components/Referral_comp.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      setcopySuccess('copied!')
+      setTimeout(() => setcopySuccess(null), 2000)
+    } catch (error) {
+      setcopySuccess('try again')
+      setTimeout(() => setcopySuccess(null), 2000)
+    }
+  }
+
+  return (
+    <div className='p-4 md:p-8 border border-solid space-y-5 md:space-y-9 rounded-3xl border-secondary w-full max-w-[950px] bg-white shadow-[0px_4px_4px_0px_rgba(77,144,219,0.21)] text-black text-base md:text-xl tracking-normal'>
```
Comment (annonymousauthority) [inline Developers/app/components/Referral_comp.js:27]: width not required

File: Developers/app/components/Referral_comp.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+      <div>
+        <h2 className='text-2xl font-semibold md:text-3xl'>Refer a friend</h2>
+        <p>
```
Comment (annonymousauthority) [inline Developers/app/components/Referral_comp.js:30]: Fonts thicker than it is in design

File: Developers/app/components/Referral_comp.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        </p>
+      </div>
+      <div>
+        <h3 className='text-lg font-medium md:text-2xl'>How it works</h3>
+        <ul className='pl-3 font-light list-disc list-inside'>
+          <li>Give your unique referral link to your friends.</li>
+          <li>Any successful referrals earn you a $150 bonus</li>
+        </ul>
+      </div>
+      <div className='flex flex-col items-center justify-between space-x-1 text-base font-medium text-black md:flex-row md:text-lg lg:text-xl'>
```
Comment (annonymousauthority) [inline Developers/app/components/Referral_comp.js:42]: 1. Equal space between referral link and link not in design.
2. Hovering on URL gives and underline.
3. On mobile the format of texts is `Referral link`, link below aligned to the left and copy by the side of link. making sure the link is truncated if passing a certain width. Referr to design if confused

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
Reduce font weight of the text as shown in the design

File: Developers/app/components/Referral_comp.js (hunk 1 above)
Comment (twikista) [reply Developers/app/components/Referral_comp.js]: width removed

File: Developers/app/components/Referral_comp.js (hunk 3 above)
Comment (twikista) [reply Developers/app/components/Referral_comp.js]: component refactored to meet design specifications

**Comment by twikista** [comment]
@annonymousauthority 
The screen recorder makes the font looks bold. Also, the difference between the design and the implementation may be due to the difference in font family used in the design (Inter) and code implementation (Lexend). please see screenshots

chrome-capture-2023-8-25 (2)
download
download (1)

**Review by annonymousauthority — APPROVED** [review]
Seen

**Pull request merged**